Parsers/ASimNetworkSession/ProductParsers/ASimNetworkSessionPaloAltoPanOS.yaml

```diff
@@ -0,0 +1,118 @@
+      , NetworkConnectionHistory=FlexString1 // Flags
+      // , UrlCategory=DeviceCustomString2 -- websessions
+      // , LogProfile=DeviceCustomString6  
+      //, VirtualSystem=DeviceCustomString3
+    // Proxy
+    // , UrlOriginal=RequestURL
+    // , HttpUserAgentOriginal=RequestClientApplication
+    // , HttpContentType=RequestContext
+    // , HttpRequestMethod=RequestMethod
+  | extend
+      NetworkBytes=tolong(FlexNumber1)
+      , SrcUsernameType="UPN"
+      , DstUsernameType="UPN"
+      , TimeGenerated  
+      , EventType="NetworkSession" // it is not clear from doc when NWConnection and when NWSession
+      , EventCount=toint(1)
+      , EventResult=case(DeviceAction=="allow","Success","Failure")
+      , NetworkSessionId=tostring(DeviceCustomNumber1)
+      , NetworkDuration=DeviceCustomNumber3
+      , EventSchemaVersion="0.2.0"
+      , EventSchema="imNetworkSession"
+      , DvcAction=case(DeviceAction=="reset client", "Reset Source",  DeviceAction=="reset server", "Reset Destination",  DeviceAction=="reset both", "Reset",  DeviceAction)
+  | project-rename
+        // DvcMacAddr=DeviceMacAddress 
+      , DstHostname=DestinationHostName
+      , DstMacAddr=DestinationMACAddress
+    , SrcMacAddr=SourceMACAddress
+      , DstIpAddr=DestinationIP
+      , DstPortNumber=DestinationPort
+      , DstNatPortNumber=DestinationTranslatedPort
+    , SrcPortNumber=SourcePort
+      , SrcIpAddr=SourceIP
+    , DstUserName=DestinationUserName
+    , SrcNatPortNumber=SourceTranslatedPort
+    , DvcOutboundInterface=DeviceOutboundInterface
+    , DvcInboundInterface=DeviceInboundInterface
+    // Trivial renames to mitigate Autocomplete
+    /// , NetworkDirection: no such data 
+    //  , EventEndTime=EndTime
+    // , EventMessage=Message
+    // parsed from additional extensions:
+        // | parse AdditionalExtensions with unparsedURL "PanOSActionFlags="ActionFlags
+        //   ";cat="Category //https://knowledgebase.paloaltonetworks.com/KCSArticleDetail?id=kA10g000000ClmHCAS
+        // ";PanOSXForwarderfor="HttpXff
+        // ";PanOSReferer="Referer 
+        //    ";"temp"l1="temp2"Gl4="temp3
+        //";PanOSVsysName="VirtualSystem
+        //";PanOSSrcUUID="SrcUUID
+        //";PanOSDstUUID="DstUUID
+        //";PanOSTunnelID="TunnelId
+        //";PanOSMonitorTag="MonitorTag
+        //";PanOSParentSessionID="ParentSessionId
+        //";PanOSParentStartTime="ParentStartTime
+        //";PanOSTunnelType="TunnelType
+        //   ";PanOSThreatCategory="ThreatCategory
+        //   ";PanOSContentVer="ContentVer
+        // for EventClassIds=="url" it parses theses additional fields  
+        //| parse temp with "PanOSXForwarderfor="HttpRequestXff";PanOSReferer="HttpReferrerOriginal	";" *
+        //| extend UrlOriginal=coalesce(UrlOriginal, unparsedURL)
+        //| project-away temp, temp2, temp3, unparsedURL
+  // -- Aliases
+  | extend
+    Dvc = DvcHostname,
+    User = SrcUsername,
```

Comment:
User is an alias to DstUsername, as you did below :-)

Reply:
:-)